ern-container-gen/src/generators/android/hull/lib/src/main/java/com/walmartlabs/ern/container/ElectrodeMiniAppActivity.java

```diff
@@ -111,4 +117,25 @@ public boolean onKeyUp(int keyCode, KeyEvent event) {
     public void onBackKey() {
         finish();
     }
+
+    @Override
+    public int checkPermission(String permission, int pid, int uid) {
```

Comment:
Should be okay to remove this method as it's returning default int value (there is no logic involved)